api/src/user/user.routes.ts

```diff
@@ -1,16 +1,11 @@
 import type express from 'express';
 import asyncHandler from 'express-async-handler';
 
-import {
-  toggleBookmark,
-  getUserWorksheet,
-  updateBookmarkBatch,
-} from './user.handlers.js';
+import { updateBookmarks, getUserWorksheet } from './user.handlers.js';
 import { authBasic } from '../auth/auth.handlers.js';
 
 export default (app: express.Express): void => {
   app.use('/api/user/*', authBasic);
-  app.post('/api/user/toggleBookmark', asyncHandler(toggleBookmark));
-  app.post('/api/user/updateBookmarkBatch', asyncHandler(updateBookmarkBatch));
+  app.post('/api/user/toggleBookmark', asyncHandler(updateBookmarks));
```

Comment:
If we are to do this, let's choose something other than "bookmark", maybe "updateWorksheet". I didn't do this because it's a slightly larger API breaking change, although it already is because of the extra `hidden` field so I'm open to doing it.

Reply:
I think let's do it, a bit annoying but makes more sense